build/openrpc/gateway.json

```diff
@@ -2,7 +2,7 @@
     "openrpc": "1.2.6",
     "info": {
         "title": "Lotus RPC API",
-        "version": "1.27.1"
+        "version": "1.27.2-dev"
```

Comment:
Why is -dev and not -rc ?

Reply:
I'm assuming from docs that getting this landed in `release/v1.27.2` is the first step, then bumping to `rc` within there is the next. @rjan90 can advise.

Reply:
will do rc1